packages/core/saved-objects/core-saved-objects-api-server-internal/src/lib/apis/internals/internal_bulk_resolve.ts

```diff
@@ -59,7 +59,7 @@ export interface InternalBulkResolveParams {
   registry: ISavedObjectTypeRegistry;
   allowedTypes: string[];
   client: RepositoryEsClient;
-  serializer: SavedObjectsSerializer;
+  serializer: ISavedObjectsSerializer;
```

Comment:
NIT: lot of methods were using the concrete type instead of the interface. Fixed all those usages